src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/Report.kt

```diff
@@ -67,8 +67,8 @@ class Report(
   var reportedBy: String,
   var reportedDate: LocalDateTime,
 
-  @ManyToOne(fetch = FetchType.LAZY, cascade = [CascadeType.ALL], optional = false)
-  val event: Event,
+  @ManyToOne(fetch = FetchType.LAZY, cascade = [CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.DETACH], optional = false)
```

Comment:
cascading delete operations from report to event meant that removing a single report resulted in _all_ sibling reports and owning event also being removed

Reply:
Why sibling reports would be deleted? That's odd 🤔

Reply:
we definitely don't want them to be. but the cascade attribute used to include cascading deletion, so when a report was deleted, the parent event was deleted and so all other reports under that were also deleted.

Reply:
now, deleting the event must be done actively rather than implicitly by cascade